python/ccxt/base/exchange.py

```diff
@@ -451,7 +451,7 @@ def fetch(self, url, method='GET', headers=None, body=None):
             self.raise_error(ExchangeError, url, method, e)
 
         except HTTPError as e:
-            self.handle_errors(response.status_code, response.reason, url, method, headers, http_response)
+            self.handle_errors(response.status_code, response.reason, url, method, headers, http_response, None)
```

Comment:
I'm not sure that this will work as intended... Because most exchanges will report a non-200 code together with the json response body containing the actual reason for it, for example, `403 + { message: "Bad signature" }`... So we should not discard the `json_response` here, I think...

- http://docs.python-requests.org/en/master/user/quickstart/#errors-and-exceptions

> `Response.raise_for_status()` will raise an HTTPError if the HTTP request returned an unsuccessful status code.

Or am I missing something?